source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -231,6 +233,49 @@ def _filter_max_successive_buses(journeys, request):
             mark_as_dead(j, "too_much_successive_buses")
 
 
+def _filter_too_much_changes(journeys, instance, request):
+    """
+    eliminates journeys with correspondences more then nb correspondence of best journey + _max_additional_changes
+    """
+    logger = logging.getLogger(__name__)
+    max_additional_changes = get_or_default(request, '_max_additional_changes',
```

Comment:
Do you prefer also to change the attribt and parameter name to "max_additional_connections" and "_max_additional_connections" ?